src/community/geogig/src/main/java/org/geogig/geoserver/wms/GeogigLayerIntegrationListener.java

```diff
@@ -235,6 +235,11 @@ public void handleRemoveEvent(CatalogRemoveEvent event) throws CatalogException
     public void reloaded() {
         // do nothing
     }
+	
+	@Override
+    public void handlePreAddEvent(CatalogBeforeAddEvent event) {
```

Comment:
The interface should provide a default implementation, backwards compatibility needs to be guarantee.

Reply:
this is an empty implemention like the above method **reloaded**